'MenuFlyoutPresenter.CornerRadius' hides inherited member 'TemplatedControl.CornerRadius'

**Describe the bug**

`CornerRadius` and `CornerRadiusProperty` need to be removed from `MenuFlyoutPresenter`. These very likely existed before `CornerRadius` was added to `TemplatedControl` and are now redundant.

```
1>C:\...\Avalonia\src\Avalonia.Controls\Flyouts\MenuFlyoutPresenter.cs(11,61,11,81): warning CS0108: 'MenuFlyoutPresenter.CornerRadiusProperty' hides inherited member 'TemplatedControl.CornerRadiusProperty'. Use the new keyword if hiding was intended.
1>C:\...\Avalonia\src\Avalonia.Controls\Flyouts\MenuFlyoutPresenter.cs(14,29,14,41): warning CS0108: 'MenuFlyoutPresenter.CornerRadius' hides inherited member 'TemplatedControl.CornerRadius'. Use the new keyword if hiding was intended.
```

**To Reproduce**
Steps to reproduce the behavior:

1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**

- OS: [e.g. Windows, Mac, Linux (State distribution)]
- Version [e.g. 0.10.0-rc1 or 0.9.12]

**Additional context**
Add any other context about the problem here.

--- a/src/Avalonia.Controls/Flyouts/MenuFlyoutPresenter.cs
+++ b/src/Avalonia.Controls/Flyouts/MenuFlyoutPresenter.cs
@@ -8,15 +8,6 @@ namespace Avalonia.Controls
 {
     public class MenuFlyoutPresenter : MenuBase
     {
-        public static readonly StyledProperty<CornerRadius> CornerRadiusProperty =
-            Border.CornerRadiusProperty.AddOwner<FlyoutPresenter>();
-
-        public CornerRadius CornerRadius
-        {
-            get => GetValue(CornerRadiusProperty);
-            set => SetValue(CornerRadiusProperty, value);
-        }
-
         public MenuFlyoutPresenter()
             :base(new DefaultMenuInteractionHandler(true))
         {

